storage/postgres/storage.go

```diff
@@ -465,6 +465,10 @@ func (ps *Storage) UpdateLabels(ctx context.Context, objectType types.ObjectType
 	return ps.updateLabels(ctx, objectType, objectID, labelChanges)
 }
 
+func (ps *Storage) GetEntitiesByTableNameMap() map[string]string {
```

Comment:
this is a bit low level, both in terminology and implementation - you are returning an internal object of the underlying scheme, what happens if someone makes modifications to it? I think we should have something like "GetEntities" that will return []Entity (new type that will have the required fields - entity name, table name etc.). the impl. can use this map as input to create the entity list.

Reply:
You're right! good point.
I've fixed it according to your suggestion.
Thanks!!!